client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -0,0 +1,211 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import QueryJetpackModules from 'components/data/query-jetpack-modules';
+import QueryJetpackSettings from 'components/data/query-jetpack-settings';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isModuleActive, isFetchingModules } from 'state/jetpack/modules/selectors';
+import {
+	getJetpackSettings,
+	isRequestingJetpackSettings,
+	isUpdatingJetpackSettings
+} from 'state/jetpack/settings/selectors';
+import { fetchSettings, updateSettings } from 'state/jetpack/settings/actions';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import ClipboardButtonInput from 'components/clipboard-button-input';
+
+class PublishingTools extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = this.buildRegeneratingState( false );
+
+		this.refreshSettings = this.refreshSettings.bind( this );
+		this.onRegenerateButtonClick = this.onRegenerateButtonClick.bind( this );
+	}
+
+	buildRegeneratingState( regenerating = false ) {
+		return {
+			regenerating
+		};
+	}
+
+	refreshSettings() {
+		const { selectedSiteId } = this.props;
+
+		this.props.fetchSettings( selectedSiteId ).then( () => {
+			this.setState( this.buildRegeneratingState( false ) );
+		} );
+	}
+
+	onRegenerateButtonClick() {
+		const { selectedSiteId } = this.props;
+
+		this.setState( this.buildRegeneratingState( true ) );
+
+		this.props.updateSettings( selectedSiteId, {
+			post_by_email_address: 'regenerate'
+		} ).then( this.refreshSettings );
+	}
+
+	isFormPending() {
+		const {
+			fetchingSettings,
+			fetchingModuleData,
+			submittingForm,
+			updatingSettings
+		} = this.props;
+
+		return fetchingSettings || fetchingModuleData || submittingForm || updatingSettings || this.state.regenerating;
+	}
+
+	renderHeader() {
+		const {
+			onSubmitForm,
+			submittingForm,
+			translate
+		} = this.props;
+
+		return (
+			<SectionHeader label={ translate( 'Publishing Tools' ) }>
```

Comment:
Alternate string suggestion: 
* `translate( 'Additional tools that make creating and publishing your content easy:' )`&mdash; translations: **3**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-settings-publishing-tools-card)